locales-src/asrouter.ftl

```diff
@@ -90,17 +90,42 @@ cfr-whatsnew-button =
   .label = What’s New
   .tooltiptext = What’s New
 
+cfr-whatsnew-fx70-hero-title = { -brand-short-name } now fights harder for your privacy
+cfr-whatsnew-fx70-hero-body = The latest update enhances the Tracking Protection feature and makes it easier than ever to create secure passwords for every site.
+
+cfr-whatsnew-fx70-notes-body = Protection reports, Secure Proxy for public Wifi, improved account controls, and many more safety and security tools are included in the latest update.
```

Comment:
Looks like "Wi-Fi" is the correct way: https://searchfox.org/mozilla-central/search?q=wifi&path=dtd

Reply:
Secure Proxy is US only, why are we exposing this to all locales?

Reply:
Ah good point. If this is supposed to be en-US/US only, we can maybe just provide the raw string in the remote message. I'll check generally what messages are supposed to be for all locales vs subset / regions.

Reply:
The "release notes" is gone with the "hero" becoming a regular message with a link to unlocalized release notes.